.github/workflows/update-latest-temporal.yml

```diff
@@ -1,37 +0,0 @@
-name: Update latest tag
-
-permissions: {}
-
-on:
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: "The tag to use as latest"
-        required: true
-
-jobs:
-  retag-and-release:
-    name: "Update latest tag"
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash
-
-    steps:
-      - name: Copy images
-        env:
-          USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          PASSWORD: ${{ secrets.DOCKER_PAT }}
-          IMAGES: server auto-setup admin-tools
-          REPO: temporalio
-          SRC_TAG: ${{ github.event.inputs.tag }}
-        run: |
-          skopeo() {
-            docker run --rm -i quay.io/skopeo/stable "$@"
-          }
-
-          for image in ${IMAGES}; do
-            src="docker://${REPO}/${image}:${SRC_TAG}"
-            dst_latest="docker://${REPO}/${image}:latest"
-            skopeo copy --dest-creds="$USERNAME":"$PASSWORD" --all "$src" "$dst_latest"
-          done
```

Comment:
This workflow has never been executed. See https://github.com/temporalio/docker-builds/actions/workflows/update-latest-temporal.yml